src/test/integrasjonstester/kotlin/no/nav/familie/ba/sak/kjerne/tilbakekreving/TilbakekrevingServiceTest.kt

```diff
@@ -35,7 +40,8 @@ class TilbakekrevingServiceTest(
     @Autowired private val tilbakekrevingRepository: TilbakekrevingRepository,
     @Autowired private val vedtaksperiodeService: VedtaksperiodeService,
     @Autowired private val databaseCleanupService: DatabaseCleanupService,
-    @Autowired private val brevmalService: BrevmalService
+    @Autowired private val brevmalService: BrevmalService,
+    @Autowired private val brevmottakerRepository: BrevmottakerRepository,
 ) : AbstractSpringIntegrationTest() {
 
     @BeforeAll
```

Comment:
En quickfix for å hindre at testene går i beina på hverandre kan være å endre denne til BeforeEach
```suggestion
    @BeforeEach
```

Reply:
haha bra spotta! Takk :D